### DISCUSSION TITLE
deleted member

### DISCUSSION THREAD

**Discussion by f14m07** [liedekef/events-made-easy#316] [Q&A]
When a member is deleted from a membership, it is permanently deleted, as are the values in the custom fields.
As there's no bin for members, I'm looking for a way to save the member and their custom field values before they're permanently deleted.
What would be the best way to do this?

**Comment by liedekef** [comment]
Set the status calculation to "manual" and the status itself to "expired", that keeps the member.
Or use the action hook eme_delete_member_action (which was not documented, but is now)

**Comment by f14m07** [comment]
Thanks for the response.
In the doc of the hook, it's writed 'executed after a member is removed'. Isn't it 'before' rather 'after'? because after the data are deleted?

**Reply by liedekef** [reply]
yes indeed, it is executed just before the member is removed (doc corrected, was a copy/paste error)

**Reply by f14m07** [reply]
thanks for this hook
